pleskdistup/actions/grub.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023-2024. WebPros International GmbH. All rights reserved.
+import os.path
+
+from pleskdistup.common import action, dist, util
+
+DEBCONF_CMD = "/usr/bin/debconf-show"
+
+class AssertGrubInstallDeviceExists(action.CheckAction):
+    def __init__(self) -> None:
+        self.name = "check GRUB installation device exists"
+        self.description = """Grub's install-device is not found: {}
+This could fail conversion - please fix it.
```

Comment:
Can we propose a solution to fix this problem here, ideally using a bunch of CLI commands?

Reply:
I added 
```
dpkg --configure grub-pc
```